dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetVersionDifference.java

```diff
@@ -500,7 +500,7 @@ private void addToNoteSummary(DatasetField dsfo, int added, int deleted, int cha
         summaryDataForNote.add(noteArray);
     }
 
-    private boolean compareFileMetadatas(FileMetadata fmdo, FileMetadata fmdn) {
+    private boolean areFileMetadatasEqual(FileMetadata fmdo, FileMetadata fmdn) {
```

Comment:
according to dictionary, word 'metadata' is uncountable which means 'metadatas' doesn't exist. I think it would be a good practice to avoid it as much as possible. In this case areFilesMetadataEqual has more sense.